### DISCUSSION TITLE
Modification - Roboto Strong variant

### DISCUSSION THREAD

**Discussion by alexcanning** [Legal-and-General/canopy#724] [Canopy requests]
**Type of request**
Modification

**Component, pattern or foundation?**
Foundation

**Nature of request**
There current strong variant of our productive type scale (Roboto) isn't bold enough. Propose changing strong from Medium 500 to Bold 700.

This change should only be applied to body text, so we need to consider how we make this change without affecting components such as buttons or input labels, which use the current Medium 500 weight.

**Comment by elenagarrone** [comment]
@alexcanning do you still want the 500 medium weight for buttons, labels, etc?
Someone will need to provide the `woff` and `woff2` files for the new Roboto 700 weight.

FYI Google only supplies `ttf` format.

If you want to keep both we should probably run quickly through the existing components to see where you want the 700 to be applied.

**Reply by alexcanning** [reply]
It will depend on the component. Buttons look ok with the medium weight but input labels could do with the extra boldness.

Looking back through the type scale, I've just realised that we also adjust the weight according to size (larger sizes are lighter) so probably need to put some thought into standardising that

**Reply by james-hayden-ellis** [reply]
@alexcanning - We could also potentially do with adding a lighter variation of Lyon to our Expressive styles too
